src/components-examples/cdk/table/cdk-table-recycle-rows/cdk-table-recycle-rows-example.css

```diff
@@ -0,0 +1,7 @@
+table {
+  width: 100%;
+}
+
+th {
+  text-align: left;
+}
```

Comment:
Could you add CSS classes to the example like `.example-table` and `.example-row` and use those here? I like to make it unambiguously clear where each styles comes from in examples so that anyone inspecting the example can quickly distinguish between example styles and component styles.

(similar for other examples)

Reply:
Done. Updated here and in the mat table demo.